OpenOversight/app/utils/forms.py

```diff
@@ -101,7 +101,7 @@ def add_officer_profile(form: AddOfficerForm, current_user: User):
                     user_id=current_user.get_id(),
                     officer=officer,
                     created_at=datetime.datetime.now(),
```

Comment:
The `last_updated_by` field should be added to these models.

Nit: You could delete the `created_at` fields since they auto-populate. The same would be fine for the `last_updated_at` field.